tests/phpunit/tests/functions/deleteoptionfreshsite.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
The filename for this test file should be `deleteOptionFreshSite.php`.

Reply:
done